src/distributions/bernoulli.rs

```diff
@@ -48,14 +48,14 @@ impl Bernoulli {
     ///
     /// For `p = 1.0`, the resulting distribution will always generate true.
     /// For `p = 0.0`, the resulting distribution will always generate false.
-    /// Due to the limitations of floating point numbers, not all internally
-    /// supported probabilities (multiples of 2<sup>-64</sup>) can be specified
-    /// using this constructor. If you need more precision, use
-    /// `Bernoulli::from_int` instead.
+    ///
+    /// This method is accurate for any input `p` in the range `[0, 1]` which is
+    /// a multiple of 2<sup>-64</sup>. If you need more precision, use `Uniform`
+    /// and a comparison instead. (Note that not all multiples of
```

Comment:
Sorry, why use `Uniform`, and what did you have in mind (given that this can generate ints and floats)? I don't think we'll include high-precision sampling code in `Uniform` anyway. Ultimately I'd say here to use `HighPrecisionFP` or whatever we get, but it's not available yet.

Reply:
Ah, for fractional probabilities. Okay, but not everything's a fraction (try π/4). Personally I think just drop this bit?